gatsby-site/src/components/Footer/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Link from 'gatsby-link';
+
+import DoiLogo from "../../img/DOI-2x.png";
+import DownloadIcon from '-!svg-react-loader!../../img/svg/icon-download-base.svg';
+
+const Footer = ({ contactInfo, siteMetadata }) => ( 
+<footer className="footer">
+  <div className="container-page-wrapper">
+    <div className="footer-col_left">
+      <a href="https://doi.gov"><img className="footer-image" src={DoiLogo} alt="Department of the Interior logo" /></a>
+    </div>
+
+    <div className="footer-col_right">
+
+      <div className="footer-bottom-left">
+	      <p className="footer-para_callout footer-para_callout-bigger">Built in the open</p>
+	      <p className="footer-para">This site (<a href={"https://github.com/18F/doi-extractives-data/releases/"+siteMetadata.version } className="link-active-beta">{ siteMetadata.version }</a>) is powered by <a className="link-active-beta" href={ siteMetadata.url+"/downloads"}>open data</a> and <a className="link-active-beta" href="https://github.com/ONRR/doi-extractives-data/">source code</a>. We welcome contributions and comments on <a className="link-active-beta" href="https://github.com/ONRR/doi-extractives-data/issues/new">GitHub</a>.</p>
+
+        <p className="footer-para-small footer-para_last"><a href="https://www.doi.gov/" className="link-beta">Department of the Interior</a> | <a href="https://www.doi.gov/privacy" className="link-beta">Privacy Policy</a> | <a href="https://www.doi.gov/foia" className="link-beta">FOIA</a> | <a href="https://www.usa.gov/" className="link-beta">USA.gov</a></p>
```

Comment:
@master12 You may want to update the footer component paths to the new GitHub url with ONRR as the org.